src/Kunstmaan/FixturesBundle/Resources/config/services.yml

```diff
@@ -13,6 +16,7 @@ services:
             - '@kunstmaan_fixtures.populator.populator'
             - '@kunstmaan_utilities.slugifier'
             - '@kunstmaan_node.pages_configuration'
+            - '%kunstmaan_fixtures.user.class%'
```

Comment:
I would not create a new parameter but use the `fos_user.model.user.class` parameter instead since that will be the one you need to configure when you want to change the default `User` object.